sopel/plugins/rules.py

```diff
@@ -1319,13 +1329,11 @@ def from_callable(cls, settings, handler):
         return cls(**kwargs)
 
     def __init__(self, nick, name, nick_aliases=None, aliases=None, **kwargs):
-        super().__init__([], **kwargs)
+        super().__init__(name, aliases=aliases, **kwargs)
         self._nick = nick
-        self._name = name
         self._nick_aliases = (tuple(nick_aliases)
                               if nick_aliases is not None
                               else tuple())
```

Comment:
Oof, I didn't notice before that every `NickCommand` is storing the bot's nick and aliases. Nothing to be done about it here, though. Don't even want to think about the kind of gymnastics that would be required to get rid of these attributes and the constructor parameters…

Reply:
Always has been. 😁 

After all, you *need* the bot's nick and its aliases from configuration in order to have `BotName: command trigger`. What bugs you about that?

Reply:
Just seems like something that should be handled at a different layer, but as I said… it'd take some hoop-jumping, and offhand I don't see a way to do it without some annoying bidirectional references (bot <-> rule manager <-> rule).

Not worth it at the moment. Might be different if we decided to support runtime changes of the bot's nick.